### PR TITLE
Copy paste enhance

### PR DESCRIPTION
**Pull request by duncanmeech**
Card has story: https://egf-atgc.atlassian.net/secure/RapidBoard.jspa?rapidView=1&view=detail&selectedIssue=EGFAD-289

### CODE REVIEW

File: src/containers/GlobalNav.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ import { undo, redo } from '../store/undo/actions';
 import { uiShowGenBankImport } from '../actions/ui';
 import { setItem } from '../middleware/localStorageCache';
 import { uiShowDNAImport } from '../actions/ui';
+import {
+  copyBlocksToClipboard
```
Comment (maxbates) [inline src/containers/GlobalNav.js]: could this be an action?

File: src/containers/uiapi.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { clipboardSetData } from '../actions/clipboard';
```
Comment (maxbates) [inline src/containers/uiapi.js]: let's get containers / components folders consistent. This isn't a component and should go somewhere else.
Idea between the two was that containers were linked to the store, components were purely driven by properties. Maybe we should pst-fix containers e.g. Inventory-container.js... but anyway, this should go into another folder. Why aren't these just actions?

Comment (duncanmeech) [inline src/containers/uiapi.js]: moved to utils/ui

File: src/containers/GlobalNav.js (hunk 1 above)
Comment (duncanmeech) [inline src/containers/GlobalNav.js]: Just an experiment that didn't work out. It is an action, that method was not used.

**Pull request merged**